AMBuilder

```diff
@@ -22,7 +22,8 @@ libsafetyhook.sources += AddSourceFilesFromDir(os.path.join(builder.currentSourc
 ])
 
 for compiler in SafetyHook.all_targets:
-  binary = libsafetyhook.Configure(compiler, libsafetyhook.name, 'Release - {0}'.format(compiler.target.arch))
+  tag = 'Debug' if builder.options.debug == '1' else 'Release'
```

Comment:
Should we decide whether or not we're in debug through a function call on `SafetyHook` structure ?
I'm mainly thinking of subprojects that need to include safetyhook for CDetour and might want some control over this. Granted, we're already nuking any preset compiler flags, so why should we care for debug but still.

Reply:
Sure, this would require them to have a `debug` option in their configure.py which is a common way to enable debug builds though. How would that EnableDebug function be used?

Reply:
Ideally I'm thinking something like
<https://github.com/alliedmodders/sourcemod/blob/master/AMBuildScript#L589>
```python
  @property
  def debug(self):
    return builder.options.debug == '1'
```
Or perhaps even reuse the `tag` property ?